assets/src/application/ui/input/SelectInput/index.ts

```diff
@@ -1 +1,5 @@
+import { OptionsType as RCOptionType } from 'rc-select/lib/interface/index';
```

Comment:
`rc-select` is a dependency of antd, should we add it explicitly(in case we will update antd, but will want to use older version of `rc-select`)?

Reply:
Will have to think about it, I wanted to copy its definition over.